code/modules/mob/living/living.dm

```diff
@@ -777,6 +777,38 @@ Thanks.
 
 	var/mob/living/L = usr
 
+	//Escaping from within a subspace tunneler.
```

Comment:
Is there a better way to identify whether the resisting mob is in the contents of a specific tunneler, or in the contents of another object that's in the contents of the tunneler?

Recursive contents checking was the only way I could think of.